cypress.config.js

```diff
@@ -18,14 +18,12 @@ module.exports = defineConfig({
   env: {
     opensearch: 'localhost:9200',
     opensearchDashboards: 'localhost:5601',
-    security_enabled: true,
+    security_enabled: false,
```

Comment:
This security config should be disabled by default because:
- Current testing workflow is fully without security
- I was checking our commands setup, and I saw that we have overwritten the request command based on this config:
  - when this config enabled, every single call of the request will be assembled with this `security_tenant=private` option. This is kinda impact the assertion we are having for now. For example, the expected url should not contain the above tenant option.